ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java

```diff
@@ -261,6 +261,23 @@ private void handleRunScript(final Command command) {
           ksqlConfig.overrideBreakingConfigsWithOriginalValues(command.getOriginalProperties()),
           overriddenProperties
       );
+      final long numPersistentQueries =
+          queryMetadataList.stream().filter(md -> md instanceof PersistentQueryMetadata).count();
+      if (!ksqlEngine.hasCapacityForPersistentQueries(numPersistentQueries, ksqlConfig)) {
```

Comment:
Couple of questions:
    - Is the reason for checking this here (instead of from start()) so that we either succeed or fail all the statements? 
    - Why not just use ```checkPersistentQueryCapacity``` - it already has the error message stuff.